param/parameterized.py

```diff
@@ -2309,12 +2382,46 @@ def params(self_, parameter_name=None):
 
     def update(self_, arg=Undefined, /, **kwargs):
         """
-        For the given dictionary or iterable or set of param=value
-        keyword arguments, sets the corresponding parameter of this
-        object or class to the given value.
+        Updates one or more parameters of this object or class.
+
+        This method allows you to set the parameters of the object or class using a dictionary,
+        an iterable, or keyword arguments in the form of param=value. The specified parameters
+        will be updated to the given values.
+
+        This method can also be used as a context manager to temporarily set and then reset
+        parameter values.
+
+        Reference: https://param.holoviz.org/user_guide/Parameters.html#other-parameterized-methods
+
+        Examples:
```

Comment:
Shouldn't this be formatted as follows?

```suggestion
        Examples
        --------
```